runtime/j9vm/javanextvmi.cpp

```diff
@@ -377,6 +375,8 @@ virtualThreadMountEnd(JNIEnv *env, jobject thread)
 				J9VMJDKINTERNALVMCONTINUATION_VMREF(currentThread, continuationObj));
 	}
 
+	VM_VMHelpers::virtualThreadHideFrames(currentThread, JNI_FALSE);
```

Comment:
Moved it closer `exitVThreadTransitionCritical`, which is closer to the end phase.